command-line.dylan

```diff
@@ -191,10 +191,11 @@ define method run-test-application
       with-open-file(stream = pathname,
                      direction: #"output",
                      if-exists: #"overwrite")
-        report-function(result, stream);
+        let cs = colorize-stream(stream);
+        report-function(result, cs);
```

Comment:
It is there because now that report function expects a coloring stream due to writing text attributes to the stream.  It would probably be better if the callee wrapped the stream that it was given instead of what I did here.